application/src/Api/Representation/AbstractResourceEntityRepresentation.php

```diff
@@ -537,7 +555,15 @@ public function displayDescription($default = null)
             $descriptionTerm = 'dcterms:description';
         }
 
-        $description = (string) $this->value($descriptionTerm, ['default' => $default]);
+        if ($lang !== null) {
+            if ($descriptionValue = $this->value($descriptionTerm, ['default' => $default, 'lang' => $lang])) {
+                $description = $descriptionValue->value();
```

Comment:
Same comment about casting vs `->value()` here.

Reply:
OK updated in 81ec357